src/rez/rezconfig.py

```diff
@@ -534,6 +534,19 @@
     "*PATH"
 ]
 
+# Some shells may require multiple types of pathing, so this option provides
```

Comment:
I feel like this could be clarified and simplified. In its current form, it's kind of hard to understand what it really does.